creator-node/src/routes/audiusUsers.js

```diff
@@ -104,7 +104,7 @@ module.exports = function (app) {
 
       await transaction.commit()
       triggerSecondarySyncs(req)
-      return successResponse({ audiusUserUUID: audiusUser.audiusUserUUID })
+      return successResponse()
```

Comment:
i think this may require a corresponding libs change?

Reply:
i don't believe this is ever used in libs. the /audius_users route is called associateCreator in libs and it's used in 3 places in this file (https://github.com/AudiusProject/audius-protocol/blob/a46fec839ff44c880ab596032c5a44296ff5b089/libs/src/api/user.js#L277). none of them ever parse the return data

Reply:
you right 👍